src/apiCalls.js

```diff
@@ -4,8 +4,18 @@ export const getResults = (coords) => {
     .catch(error => console.error(error))
 }
 
+//If we need a proxy, here is the function we should use: 
```

Comment:
Thanks for including this commented code for quick access if we end up needing the proxy!